src/cache/cache.ts

```diff
@@ -321,7 +321,7 @@ export interface CacheInstance {
    * @default defaultResponseInterceptor
    * @see https://axios-cache-interceptor.js.org/config#responseinterceptor
    */
-  responseInterceptor: AxiosInterceptor<CacheAxiosResponse<unknown, unknown>>;
+  responseInterceptor: AxiosInterceptor<AxiosResponse<unknown, unknown>>;
```

Comment:
would be great if you could keep the `CacheAxiosResponse` type here.

Reply:
Unfortunately changing this cascades up to the initial problem : the definition of `AxiosCacheInstance`.
Can you elaborate on why you want this `responseInterceptor` type to be using `CacheAxiosResponse`, it may help me understand what are my options ?